kotlin-source/src/main/kotlin/com/example/state/IOUState.kt

```diff
@@ -31,10 +30,10 @@ data class IOUState(val iou: IOU,
         LinearState, QueryableState {
 
     /** The public keys of the involved parties. */
-    override val participants: List<CompositeKey> get() = listOf(sender, recipient).map { it.owningKey }
+    override val participants: List<PublicKey> get() = listOf(sender, recipient).map { it.owningKey }
 
     /** Tells the vault to track a state if we are one of the parties involved. */
-    override fun isRelevant(ourKeys: Set<PublicKey>) = ourKeys.intersect(participants.flatMap {it.keys}).isNotEmpty()
+    override fun isRelevant(ourKeys: Set<PublicKey>) = ourKeys.intersect(participants).isNotEmpty()
```

Comment:
The same comment as above applies here. Should be: `ourKeys.intersect(participants.flatMap {it.keys}).isNotEmpty()`